src/vorta/views/repo_add_dialog.py

```diff
@@ -211,6 +214,11 @@ def password_listener(self):
             self.passwordLabel.setText(translate('utils', msg))
             return not bool(msg)
 
+    def cancel_job(self):
+        if self.job:
+            self.job.cancel()
+            logger.info('Cancelled adding repo.')
```

Comment:
If you keep it, it should be of level `debug`.

Reply:
👍🏼